R/Model-class.R

```diff
@@ -2284,11 +2284,11 @@ EffFlexi <- function(eff,
   D1 <- cbind(0, diag(data@nGrid - 1)) - cbind(diag(data@nGrid - 1), 0)
   if (rw1) { # the rank-deficient prior precision for the RW1 prior.
     RW <- crossprod(D1)
-    RW_rank <- data@nGrid - 1L # rank = dimension - 1.
+    RW_rank <- data@nGrid - 1L # rank = dimension - 1. # nolintr
```

Comment:
I think here it would be nice to update to `rw` etc. since that does not need any changes elsewhere?

Reply:
yah, just wanted to keep upper case for matrices, since RW is a matrix, also mentioned in the help, otherwise it might be mixed with `rw1` flag we use also here.....

Reply:
ok up to you.. also `rw_mat` could be an option. basically the more we can minimize `# nolintr` usage the easier for maintenance as well

Reply:
Thanks @danielinteractive , I agree. Maybe in this case I would leave `RW` as it corresponds with the name of the slot in the corresponding class, so I would need to change the class itself. Maybe it would be also a good idea but we then have other variables-matrices, where we do not use `_mat` suffix, and these come from the equation and etc. So I would just leave it here as it is.